en/docs/integrate/develop/working-with-service-catalog.md

```diff
@@ -0,0 +1,81 @@
+# Working with Service Catalog
+
+Service Catalog is one of the main attributes which enables the API-first Integration in WSO2 API Manager. Through the Service Catalog, integration services are made discoverable to the API Management layer so that API proxies can directly be created using them.
+
+## Getting Started with Service Catalog
+
+1. Go to the **Publisher Portal** (https://localhost:9443/publisher)
+2. Click on the **Hamburger Icon** on the top left corner of the page.
+3. Select the **Service Catalog** Menu as follows.
+
+![Service Catalog Menu](../../assets/img/integrate/service-catalog/go-to-service-catalog.png)
```

Comment:
```suggestion
![Service Catalog Menu]({{base_path}}/assets/img/integrate/service-catalog/go-to-service-catalog.png)
```

Please add base_path for all links. Relative links can break sometimes if we move content around so we use this approach instead. Please fix in other instances.